src/patternfly/components/Dropdown/dropdown-menu.hbs

```diff
@@ -4,10 +4,48 @@
   {{#if dropdown-menu--attribute}}
 	  {{{dropdown-menu--attribute}}}
   {{/if}}>
-  <li><a class="pf-c-dropdown__menu-item" href="#">Link</a></li>
-  <li><button class="pf-c-dropdown__menu-item">Action</button></li>
-  <li><a class="pf-c-dropdown__menu-item pf-m-disabled" aria-disabled="true" tabindex="-1" href="#">Disabled link</a></li>
-  <li><button class="pf-c-dropdown__menu-item" disabled>Disabled action</button></li>
+  <li>
+    {{#if dropdown--HasItemIcons}}
+      {{#> dropdown-menu-item dropdown-menu-item--type="a" dropdown-menu-item--modifier="pf-m-icon" dropdown-menu-item--attribute='href="#"'}}
+        {{#> dropdown-menu-item-icon}}
+          <img src="/assets/images/pf-logo-small.svg" alt="">
```

Comment:
needs `aria-hidden` val

Reply:
from https://github.com/patternfly/patternfly-next/pull/1916#discussion_r293372913

> Since you are using `<img>` elements for these icons, AND since these images are redundant (i.e. the text that follows is considered synonymous with these icons), then having `alt=""` like you have it here is the correct way to tell assistive technology that this image provides no extra meaning and to ignore it.

Reply:
Still, doesn't pass axe testing. If @jgiardino ok'd it in the past, I'm good with it.